tensorboard/compat/tensorboard/BUILD

```diff
@@ -0,0 +1,22 @@
+# Description:
```

Comment:
Instead of `compat/tensorboard` can we do `compat/writer`? @nfelt what are your thoughts on the right naming for this?